lib/babe/build.go

```diff
@@ -252,6 +252,18 @@ func buildBlockInherents(slot Slot, rt runtime.Instance) ([][]byte, error) {
 		return nil, err
 	}
 
+	parachainInherent := ParachainInherentData{}
+	// add parachn0 and newheads
+	// for now we can use "empty" values, as we require parachain-specific
+	// logic to actually provide the data.
+
+	if err = idata.SetStructInherent(types.Parachn0, parachainInherent); err != nil {
+		return nil, err
+	}
+	if err = idata.SetStructInherent(types.Newheads, []byte{0}); err != nil {
+		return nil, err
+	}
+
```

Comment:
@timwu20 does this scale encoding look good to you? The error that i am getting tells that the encoding is not right!
```
2022-07-04T10:14:05Z CRIT target=runtime message=panicked at 'Bad input data provided to inherent_extrinsics: Codec error', /home/elizabeth/substrate/bin/node/runtime/src/lib.rs:1099:1	ext_logging_log_version_1	pkg=runtime module=go-wasmer
badger 2022/07/04 10:14:05 INFO: [Compactor: 173] Running compaction: {level:0 score:1.73 dropPrefixes:[]} for level: 0
badger 2022/07/04 10:14:05 INFO: LOG Compact 0->1, del 1 tables, add 1 tables, took 1.592695ms
badger 2022/07/04 10:14:05 INFO: [Compactor: 173] Compaction for level: 0 DONE
badger 2022/07/04 10:14:05 INFO: Force compaction on level 0 done
--- FAIL: TestVerifyAuthorshipRight (8.20s)
    build_integration_test.go:97: 
        	Error Trace:	build_integration_test.go:97
        	            				verify_integration_test.go:376
        	Error:      	Received unexpected error:
        	            	cannot build inherents: Failed to call the `BlockBuilder_inherent_extrinsics` exported function.
        	Test:       	TestVerifyAuthorshipRight

```